backend/grant/proposal/views.py

```diff
@@ -19,6 +19,7 @@
 from grant.utils.enums import ProposalStatus, ContributionStatus
 from grant.utils import pagination
 from sqlalchemy import or_
+import datetime
```

Comment:
Doing `from datetime import datetime` avoids having to do `datetime.datetime` in places.

Also, you can remove the parse import now.